.drone.yml

```diff
@@ -5840,6 +5840,145 @@ steps:
+  - name: snapshots
+    host:
+      path: /usr/local/etc/tezedge-ci/data/
+  - name: tools
+    host:
+      path: /usr/local/etc/tezedge-ci/tools
+  - name: cache
+    host:
+      path: /usr/local/etc/tezedge-ci/data/cache/build_${DRONE_BUILD_NUMBER}/${DRONE_STAGE_NAME}
+
+steps:
+
+  - name: octez-node
+    image: tezos/tezos:v9-release
+    detach: true
+    environment:
+      TEZOS_NODE_DIR: /home/tezos/data
+    volumes:
+      - name: identities
+        path: /identities
+      - name: snapshots
+        path: /snapshots
+    commands:
+      - tezos-node config init --network florencenet
+      - cp /identities/identity_1.json $${TEZOS_NODE_DIR}/identity.json
+      - tezos-node snapshot import /snapshots/octez-009-florence-1000.snapshot --block BKqNZHy5CFYLD6v6BjDLrEavrkK4qH39hhrsD7QtbWcdnnwoYub
+      - tezos-node run --no-bootstrap-peers
+
+  - name: tezedge-new-node
+    image: tezedge/tezedge-ci-builder
+    user: root
+    volumes:
+      - name: build
+        path: /build
+      - name: tools
+        path: /tools
+    commands:
+      # let octez node to be up and running
+      - sleep 15
+      - OCTEZ_IP=$(perl -e 'use Socket; my $a = gethostbyname("octez-node"); print inet_ntoa($a)')
+      # use alloc instead of jemalloc by commenting out lines between these two
+      - export LD_LIBRARY_PATH=/build/ffi
+      - >
+        /build/light-node --protocol-runner /build/protocol-runner
+        --network florencenet --peers $${OCTEZ_IP}:9732
+        --config-file /build/tezedge/tezedge_drone.config
+        --identity-file /build/identities/identity_2.json --rpc-port 18732
+        --peer-thresh-low 1 --peer-thresh-high 1 --synchronization-thresh=0
+        --init-sapling-spend-params-file /build/ffi/sapling-spend.params
+        --init-sapling-output-params-file /build/ffi/sapling-output.params &
+        echo $! > /var/run/tezedge.pid
+      # this command waits the node to be on level 1000
+      - |
+        time -o tezedge-new.time -f "%e" sh -c '
+        block=0
+        while [ $block -lt 1000 ]; do
+          sleep 1
+          b=$(curl -s localhost:18732/chains/main/blocks/head | jq .header.level)
+          block=$${b:-$block}
+          echo "===> Block level $block"
+        done
+        '
+
+  - name: tezedge-old-node
+    image: tezedge/tezedge-ci-builder
+    user: root
+    volumes:
+      - name: build
+        path: /build
+      - name: tools
+        path: /tools
+    commands:
+      - OCTEZ_IP=$(perl -e 'use Socket; my $a = gethostbyname("octez-node"); print inet_ntoa($a)')
+      - git reset --hard origin/$${DRONE_TARGET_BRANCH}
+      - rm -rf ./tezos-node-data ./light-node-data
+      - cargo clean && cargo build --release --package light-node --package protocol-runner
+      - export LD_LIBRARY_PATH=tezos/sys/lib_tezos/artifacts/
+      - >
+        target/release/light-node --protocol-runner target/release/protocol-runner
```

Comment:
@akoptelov 
we should create one pipeline which builds old tezedge binary (release and/or debug),
and once build, use `synchronize_ci.sh` to copy that binary to other servers,
and all pipelines which required old binary should depends on,
just to build old tezedge once